cura/FirmwareUpdater.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2018 Ultimaker B.V.
+# Cura is released under the terms of the LGPLv3 or higher.
+
+from PyQt5.QtCore import QObject, QUrl, pyqtSignal, pyqtProperty
```

Comment:
It would be nice if this class is inside the PrinterOutput folder instead of in the root cura folder.

Reply:
It's sending something to a connected printer (just not a print job). Given that it's tightly coupled with the other printer output functionality I think it's nice to have it in the same place. The root folder is all over the place :P

Reply:
PrinterOutputDevice.py also isn't in PrinterOutput. Shall I move that too?